### ISSUE TITLE
Enumerated list of supported table types per venue

### ISSUE THREAD

**Issue by kharimames** [Alkaar/resy-booking-bot#110]
Is there a way to determine a list of table types supported by each venue?  For example, I see 'Dining Room' as one option in the documentation but I know one venue that has a main dining room as well as a private dining room, and I can imagine other venues might have different naming conventions for their rooms.  Would you happen to know if the list of available table types is static or configured by the venue; if the latter, how can we determine what that looks like?
I tried searching the `/find` endpoint response for one venue but couldn't find any that resembled a list of these types.

**Comment by Alkaar** [comment]
The `/find` endpoint response actually has the list of table types. It's hard to parse because it's in JSON format. I would recommend you copying/pasting it into a JSON formatter where it'll be easier to read. I tried to find if there's a better way to determine what the list of possible table types are and this is the only way as far as I can tell.
